packages/palette/src/elements/ModalDialog/ModalDialogContent.tsx

```diff
@@ -104,3 +109,13 @@ export const ModalDialogContent: React.FC<ModalDialogContentProps> = ({
     </Flex>
   )
 }
+
+const Close = styled(Clickable)`
+  transition: opacity 250ms;
```

Comment:
Don't we have this stored somewhere as a CONST?

Reply:
We don't. I don't really think we need to, unless there was some agreed upon values and guidelines for animations and transitions in the DS Figma (there probably should be) — I'm just basing values off of personal experience with what feels good. In this case it's just a focus so, actually, we should maybe not have any focus just so it's a bit more apparent.